ql/src/java/org/apache/hadoop/hive/ql/optimizer/Optimizer.java

```diff
@@ -192,7 +192,9 @@ public void initialize(HiveConf hiveConf) {
       transformations.add(new FixedBucketPruningOptimizer(compatMode));
     }
 
-    transformations.add(new BucketVersionPopulator());
+    if (!isTezExecEngine) {
```

Comment:
how is that related to the current patch?

Reply:
Mistake